results/mysql-kafka/README.md

```diff
@@ -0,0 +1,93 @@
+# MySQL-to-Kafka results
+
```

Comment:
And one last ask - please add a short entry in the main `README.md` in the root of the repo and link to the full report here for more info. Same as we already have for the other 3 benchmarks. Thanks!

Reply:
done